networkx/algorithms/operators/product.py

```diff
@@ -181,16 +183,15 @@ def tensor_product(G, H):
 def cartesian_product(G, H):
     """Return the Cartesian product of G and H.
 
-    The tensor product P of the graphs G and H has a node set that
-    is the Cartesian product of the node sets, $V(P)=V(G) \times V(H)$.
-    P has an edge ((u,v),(x,y)) if and only if (u,v) is an edge in G 
-    and x==y or  and (x,y) is an edge in H and u==v.
-    and (x,y) is an edge in H.
+    The Cartesian product P of the graphs G and H has a node set that
+    is the Cartesian product of the node sets, :math:`V(P)=V(G) \times V(H)`.
+    P has an edge ((u,v),(x,y)) if and only if either u is equal to x and
+    v & y are adjacent in H or if v is equal to y and u & x are adjacent in G.
```

Comment:
Do you know if this definition is standard?  I've typically seen it as ((u,v), (x,y)) so long as (u,x) or (v,y) is an edge. Notably, there is no restriction that u == x or v == y.